packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.stories.js

```diff
@@ -23,16 +24,6 @@ const subject = {
   ]
 }
 
-let zoomCallback
-
-function onZoom(type) {
-  zoomCallback(type)
-}
-
-function setZoomCallback(callback) {
-  zoomCallback = callback
-}
-
```

Comment:
Removing these breaks the zoom buttons in the third story with pan and zoom. They get covered up by the image itself because there isn't a layout setup to make sure that doesn't happen. The stories could be updated to better reflect the image toolbar usage. Maybe actually import it and connect it if we're going to mock the store.

Reply:
I think the button layout is broken on master. Mark commented on it recently.

Reply:
I've fixed the layout by moving the buttons above the subject viewer. I've also imported the connected buttons so that they use the `subjectViewer` model (and work continuously on pointer down.)

Reply:
I've also added keyboard shortcuts to those buttons.

Reply:
Looks good. Thanks!